src/Components/Samples/BlazorUnitedApp/Program.cs

```diff
@@ -8,6 +8,7 @@
 
 // Add services to the container.
 builder.Services.AddRazorComponents();
+builder.Services.AddAntiforgery();
```

Comment:
What's the reasoning behind this change and the removal on line 28?

Reply:
This is a reaction to anti-forgery changes that were made in https://github.com/dotnet/aspnetcore/pull/49530.

Since Blazor requires anti-forgery validation on all components, the endpoint routing middleware will present an exception to the user if no anti-forgery middleware is enabled to validate the token.

The `WebApplicationBuilder` will automatically enable the anti-forgery middleware if anti-forgery services are registered which is why I added this line of code.

I removed `UseRouting` because the `WebApplicationBuilder` also automatically enabled this. There's usually no reason to call it yourself unless you want to change the order of middleware in your application. 

[Here is the code](https://github.com/dotnet/aspnetcore/blob/c2d95a80e60c20dd6ec9a7debc522c47782b2bf5/src/DefaultBuilder/src/WebApplicationBuilder.cs#L416) where all the implicit middlewares are enabled.

Reply:
> The WebApplicationBuilder will automatically enable the anti-forgery middleware if anti-forgery services are registered which is why I added this line of code.

Ah, makes sense! It looks like `AddRazorComponents()` already calls `AddAntiforgery()` though - would that make this new change redundant?

https://github.com/dotnet/aspnetcore/blob/a0db11ba7c3ae217e9745976056f10cb2a7dafde/src/Components/Endpoints/src/DependencyInjection/RazorComponentsServiceCollectionExtensions.cs#L33-L38

Reply:
Indeed -- we can remove it altogether. Good catch!